lib/inspector-window.js

```diff
@@ -82,7 +83,9 @@ const InspectorWindow = Class(
     }
 
     let url = INSPECTOR_XUL_URL;
-    return this.open("rdp-inspector", url);
+    return this.open("rdp-inspector", url, [{
+      inspectorWindowName: this.inspectorWindowName
+    }]);
   },
```

Comment:
inspectorWindowName isn't set when 'standard' Toolbox's connection is observed.
Is it a bug or a feature (it's ok for me if it's done as a follow up).